docs/user-guide/ingest-data/for-observability/opentelemetry.md

```diff
@@ -185,3 +185,59 @@ Default table schema:
 - All attributes, including resource attributes, scope attributes, and log attributes, will be stored as a JSON column in the GreptimeDB table.
 - The timestamp of the log will be used as the timestamp index in GreptimeDB, with the column name `timestamp`. It is preferred to use `time_unix_nano` as the timestamp column. If `time_unix_nano` is not provided, `observed_time_unix_nano` will be used instead.
 
+## Traces
+
+GreptimeDB supports writing OpenTelemetry traces data directly via the [OTLP/HTTP](https://opentelemetry.io/docs/specs/otlp/#otlphttp) protocol, and it also provides a table model of OpenTelemetry traces for users to query and analyze traces data conveniently.
+
+### OTLP/HTTP API
+
+You can use [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/) or other similar technologies to add traces data to your application. You can also use [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to collect traces data and use GreptimeDB as the backend storage.
+
+To send OpenTelemetry traces data to GreptimeDB through OpenTelemetry SDK libraries, please use the following information:
+
+* URL: `http{s}://<host>/v1/otlp/v1/traces`
+* Headers: The headers section is the same as the [Logs](#Logs) section, you can refer to the [Logs](#Logs) section for more information. 
+
+By default, GreptimeDB will write traces data to the `opentelemetry_traces` table in the `public` database. If you want to write traces data to a different table, you can use the `X-Greptime-DB-Name` and `X-Greptime-Log-Table-Name` headers to specify the database and table name.
+
+GreptimeDB will accept **protobuf encoded traces data** via **HTTP protocol**, and the `content-type` header should be `application/x-protobuf`.
+
```

Comment:
We need to mention `x-greptime-pipeline-name`: `"greptime_trace_v1"` is required for trace.